### DISCUSSION TITLE
Reply includes Obsidian Link

### DISCUSSION THREAD

**Discussion by anandbhaskaran** [brianpetro/obsidian-smart-connections#420] [Ideas]
At the moment system is super amazing to answer all the questions that I ask for. However, it does not point me to the document(s) from where it obtained this information. Outputting the documents as obsidian links might be a game changer.

**Comment by brianpetro** [comment]
@anandbhaskaran thanks for the feedback 😊

At one point I tried implementing footnote style references, but it was unreliable in terms of the chat accurately inserting links to the footnotes.

Something I'm considering is an "expand" button in the Chat message UI that would display the links to the context.

I do think that this particular UI/UX could be improved, so if you have any suggestions I'd be happy to hear them.

In the meantime, in the bottom right corner of the chat messages, there should be a button that copies the context used in the chat to the clipboard. You can then paste this into a note for review.

Thanks for your help in making Smart Connections the best it can be,
Brian 🌴

**Comment by heyalexnet** [comment]
I think that's references would be one of the most useful things to add. I'm currently doing what you advised in your comment (copying, creating a note and pasting the links), but it's obviously not productive nor user friendly ;-) I don't know about Obsidian UI possibilities to advise you well but what about a modal window? (not ideal but better than current solution)

**Reply by brianpetro** [reply]
A modal could work well, good idea.

🌴

**Comment by brianpetro** [comment]
@anandbhaskaran @ineation I'm moving this discussion to this issue https://github.com/brianpetro/obsidian-smart-connections/issues/422 where I keep track of feature requests like this.

I already added the key details, but please feel free to add more, as it will increase the priority of the issue in my system.

Thanks for your help!

🌴
